### ISSUE TITLE
some keymappings not working with developer version on linux

### ISSUE THREAD

**Issue by MarcusE1W** [LightTable/LightTable#2338]
Some keys do not work as defined in the keymap.

So far I have found:
- auto-complete with tab is ignored (you only get the tab)
- ctrl-d deletes the current line instead of showing documentation

This has been tested with Archlinux (git AUR package) and Linux Lite (based on Ubuntu) following the guide for a developer install.
Standard keymaps have been used for testing.

Interestingly, when you install both packages in parallel; standard release 0.8.1 from the web site and the developer version from git they share happily the same configuration files in .config/LightTable.
In the standard version auto-complete and ctrl-d work as expected in the git version they show the above mentioned unexpected behavior.

A similar problem has also been reported here: https://groups.google.com/forum/#!topic/light-table-discussion/I36BkNgvme8

### Steps to Reproduce on Linux

 1. install a linux from scratch or use an existing one ( in a VM for example)
 2. install developer version of LightTable from git
3. write something and then press ctrl-d
4. open the behaviors and try to use autocomplete

I have not tested on Windows or MacOS

**Comment by sbauer322** [comment]
You are absolutely correct. It looks like a regression was introduced in commit https://github.com/LightTable/LightTable/commit/75d65c35edab455dbca5a62250036b0f55dc51cf, which has not been in an official release yet.

**Comment by Mouvedia** [comment]
@sbauer322 could you add all regression issues to the [0.9.0 milestone](https://github.com/LightTable/LightTable/milestone/15)?

**Comment by sbauer322** [comment]
I've added two `regression` issues I feel are reasonable for the milestone - I'm not familiar enough with the lag regression issue to add it.
